samples/bookinfo/networking/egress-rule-google-apis.yaml

```diff
@@ -4,18 +4,43 @@ metadata:
   name: googleapis
 spec:
   hosts:
-  - "*.googleapis.com"
+  - www.googleapis.com
   ports:
+  - number: 80
+    name: http
+    protocol: HTTP
```

Comment:
@frankbu I think Vadim is on vacation, do you know why he adds number 80?

Reply:
i am not seeing bookinfo calls http://www.googleapis.com:80, only https

Reply:
Vadim added a flag in the source code to optionally use port 80. He's showing how you can do fine-grain monitoring of calls to external services.
https://preliminary.istio.io/blog/2018/egress-https/#tls-origination-by-istio

Reply:
Ah thanks @frankbu, sorry missed that earlier!